File: cpp/771.cpp
//
// Created by Congb on 2019/9/25.
//
#include <iostream>
#include <string>
#define ture true

using namespace std;

int numJewelsInStones(string J, string S) {
    int jl=J.length(), sl=S.length(), r=0;
    for(int p=0; p<sl; p++) {
        for(int q=0; q<jl; q++) {
            if(S[p]==J[q]) {
                r++;
                break;
            }
        }
    }
    return r;
}

int main() {
    cout << numJewelsInStones("aA", "aAAbbbb");
    return 0;
}

File: 771.js
/**
 * @param {string} J
 * @param {string} S
 * @return {number}
 */

var numJewelsInStones = function(J, S) {
    let a = {};
    let r = 0;
    let jl = J.length, sl = S.length;
    for(let p = 0; p<sl; p++) {
        for (let q =0; q<jl; q++) {
            if (S[p]==J[q]) r++;
        }
    }

    // console.log(r);
    return r;
};

console.log(numJewelsInStones(J = "aA", S = "aAAbbbb"));